lineman/app/services/ability_service.coffee

```diff
@@ -55,7 +55,7 @@ angular.module('loomioApp').factory 'AbilityService', (CurrentUser) ->
 
     canEditComment: (comment) ->
       CurrentUser.isAuthorOf(comment) and
-      comment.isMostRecent() or comment.group().membersCanEditComments
+      (comment.isMostRecent() or comment.group().membersCanEditComments)
```

Comment:
Ah, I was just confused because the PR said 'edit thread' but this is a fix for edit commit.

In any case, yay fixing things!